README.md

```diff
@@ -1 +1,30 @@
-# ruby_enumerables
\ No newline at end of file
+# Project Name
+Recreating Ruby Enumerables
+
+## Table of contents
+* [General info](#general-info)
+* [Setup](#setup)
+* [Technologies](#technologies)
+* [Features](#features)
+* [Status](#status)
+* [Contact](#contact)
+
+## General info
+The above project was undertaken so as to recreate and reconstruct Ruby iterators & enumerable methods without the use of any in-built iterators, and that would work perfectly on arrays and Hashes just like the Ruby enumerators. I created a Module that houses all my iterator methods. It is created and built by <a href="https://github.com/adaorachi">MaryAnn Chukwuka</a>.
+
+## Setup
+It is pretty simple to setup. All you have to do is to clone the project into your local repository.
+
+## Technologies
+* Ruby (Language)
+* VSCode (Text Editor)
```

Comment:
[OPTIONAL] 
I won't say this is necessary to add to your README as this specific text editor is not a necessary component to making the project work. It could easily be edited using sublime text 3 for instance.